tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts

```diff
@@ -0,0 +1,43 @@
+import { Species } from "./base";
+
+const Vampire: Species = {
+  description: "A classy Vampire! They descend upon Space Station Thirteen \
+    Every year to spook the crew! \"Bleeg!!\"",
+  features: {
+    good: [{
+      icon: "bed",
+      name: "Coffin Brooding",
+      description: "Vampires can delay The Thirst and heal by resting in a \
+        coffin. So THAT'S why they do that!",
+    }, {
+      icon: "skull",
+      name: "Minor Undead",
+      description: "Minor undead enjoy some of the perks of being dead, like \
+        not needing to breathe or eat, but do not get many of the \
+        environmental immunities involved with being fully undead.",
+    }, {
+      icon: "recycle",
+      name: "Bat Form",
+      description: "Vampires can become bats. Bats are very weak, but \
+        are great for escaping bad situations. They can also travel through \
+        vents, giving Vampires a lot of access. Just remember that access \
+        doesn't equal permission, and people may be unhappy with you showing \
+        up anywhere!",
```

Comment:
```suggestion
        up uninvited!",
```

I just think it's kinda funny considering that thing about vampires not being allowed in if they're not invited.